fibonacci.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+
+int atoi(char *number)
+{
+    int n = 0, i = 0;
+    while( *(number + i) != '\0' ) {
+
+        n = n * 10 + *(number + i) - 48;
+        ++i;
+    }
+    return n;
+}
+
+int fibonacci_rec(int n)
+{
+    if(n < 2) {
+        return 1;
+    }
+    else {
+        return fibonacci_rec(n - 1) + fibonacci_rec(n - 2);
```

Comment:
Does it make sense to have both the iterative and recursive versions in the same file? How much code do they share?